app/models/rook.rb

```diff
@@ -1 +1,9 @@
-class Rook < Piece; end
+class Rook < Piece
+  def valid_move?(destination_x, destination_y)
+    destination_x == x || destination_y == y
+  end
```

Comment:
Nice, can you add tests for it? Take a look at @kristinh12's PR for some good examples: https://github.com/TheFirehoseProject-TeamProject/ChessApp/pull/27